### ISSUE TITLE
Unable to manually call Layout in Vue 2.0

### ISSUE THREAD

**Issue by Splendorr** [David-Desmaisons/Vue.Isotope#13]
Hi there! I really appreciate this library. Thanks for your work on it.

I'm afraid I'm missing one piece, though, which is how to call the built-in "layout" method manually. What I want is to manually invoke layout from a separate function call.

I have several functioning Vue.Isotope objects on the page, and am able to interact with them in other ways. I just can't figure out how to use the Methods described in the Readme. If my Vue object is called `app`, shouldn't `app.layout('isotopeID')` work? Or is there some way to refer to a specific Vue.Isotope object that I've missed?

I feel like I'm missing something pretty basic, but nothing I've tried has worked. Any help would be greatly appreciated!

**Comment by David-Desmaisons** [comment]
Take a look at example in the example folder of the repo: [index.html](https://github.com/David-Desmaisons/Vue.Isotope/blob/master/examples/index.html).

You can use ref for child component:

```javascript
 <isotope ref="cpt" id="root_isotope" class="isoDefault" :options='getOptions()' :list="list" @filter="filterOption=arguments[0]" @sort="sortOption=arguments[0]">
    <div v-for="element in list" :key="element.id"  @click="selected=element">
        {{element.name}}
        <br>
        {{element.id}}
    </div>
</isotope>
```

And then use the built-in ```$refs``` using the corresponding name in the parent component:

```javascript
<button @click="$refs.cpt.shuffle()">Shuffle</button>
```

This example use shuffle but it could be layout of course..

**Comment by Splendorr** [comment]
Thank you. Here's what I got to work:

My site has containers that change widths. The Isotopes were inside these containers. Sometimes, the Isotopes would layout before the width of the container had finished animating, which left the isotopes stacked on top of each other.

This may not be the most performant solution, but I attached these calls to the Vue2 "update" event, in the Vue declaration. My Vue vm is called "app".

```
updated: function() {
    console.log('************** vue updated ****');
    app.$refs.mainiso.layout('fitRows');
},
```

This calls the layout method each time the Vue model is updated. Since the Vue model really only updates when something happens to the Isotopes or one of these containers changes size, this seems to work for me.

Thanks again!
